doc/sphinx-guides/source/developers/containers.rst

```diff
@@ -2,17 +2,38 @@
 Docker, Kubernetes, and Containers
 ==================================
 
-The Dataverse Community is exploring the use of Docker, Kubernetes, and other container-related technologies. The primary community-lead projects to watch are:
+The Dataverse community is exploring the use of Docker, Kubernetes, and other container-related technologies.
+
+.. contents:: |toctitle|
+	:local:
+
+Container Guide
+---------------
+
+We recommend starting with the :doc:`/container/index`. The core Dataverse development team, with lots of help from the community, is iterating on containerizing the Dataverse software and its dependencies there.
+
+Help Containerize Dataverse
+---------------------------
+
+If you would like to contribute to the containerization effort, please consider joining the `Containerization Working Group <https://dataverse.org/working-groups#cwg>`_.
+
+Community-Lead Projects
+-----------------------
+
+The primary community-lead projects (which the core team is drawing inspiration from!) are:
 
-- https://github.com/IQSS/dataverse-kubernetes
 - https://github.com/IQSS/dataverse-docker
+- https://github.com/IQSS/dataverse-kubernetes (especially the https://github.com/EOSC-synergy/dataverse-kubernetes fork)
 
-The :doc:`testing` section mentions using Docker for integration tests.
+Deprecated Projects
+-------------------
 
-See also the :doc:`/container/index`.
+The :doc:`testing` section mentions using docker-aio for integration tests. We do not plan to keep this project alive.
```

Comment:
Depreciation plan or end of life notice would be nice.

Reply:
I'm not sure I get you here. Do you want such a notice in here, right now? Or is this something you'd like to see in the future?

Reply:
Oh! Is anyone except me (occasionally) using docker-aio?!?